template/src/utils/useString.ts

```diff
@@ -0,0 +1,63 @@
+/*
+********************************************
+ Copyright © 2021 Agora Lab, Inc., all rights reserved.
+ AppBuilder and all associated components, source code, APIs, services, and documentation 
+ (the “Materials”) are owned by Agora Lab, Inc. and its licensors. The Materials may not be 
+ accessed, used, modified, or distributed for any purpose without a license from Agora Lab, Inc.  
+ Use without a license or in violation of any license terms and conditions (including use for 
+ any purpose competitive to Agora Lab, Inc.’s business) is strictly prohibited. For more 
+ information visit https://appbuilder.agora.io. 
+*********************************************
+*/
+import {useFpe} from 'fpe-api';
+import {useLanguage} from '../language/useLanguage';
+import {DEFAULT_I18_DATA} from '../language';
+import {TextDataInterface} from '../language/default-labels/index';
+
+export function usei18nData(
+  selectedLanguageCode: string = DEFAULT_I18_DATA.locale,
+) {
+  const languageData = useFpe((data) => data?.i18n);
+  if (languageData && languageData.length) {
```

Comment:
I think you want to make sure if languageData is an array, for that use Array.isArray. 
.length will exist on objects as well,

Reply:
done